cortex-m-rt/link.x.in

```diff
@@ -144,8 +144,9 @@ SECTIONS
     __veneer_base = .;
     *(.gnu.sgstubs*)
     . = ALIGN(32);
-    __veneer_limit = .;
   } > FLASH
+  . = ALIGN(32);
```

Comment:
```suggestion
  /* Place `__veneer_limit` outside the `.gnu.sgstubs` section to work around size calculation error, see #434 */
  . = ALIGN(32);
```

Reply:
Thanks! It will be nice to add this description. BTW, I may need to request for review again after this change, as it will dismiss review suggestions on GitHub.